synapse/lib/aha.py

```diff
@@ -388,6 +388,21 @@ async def fini():
         self.slab.initdb('aha:provs')
         self.slab.initdb('aha:enrolls')
 
+        netw = self.conf.get('aha:network')
+        if netw is not None:
+            # Create the root@network as an admin if it does not exist.
+            # If it does exist, ensure that user has admin permissions.
+            defprovuser = f'root@{netw}'
+            user = await self.auth.getUserByName(defprovuser)
+            if user is None:
+                await self._addAdminUser(defprovuser)
```

Comment:
This can't use non-nexusified APIs like _addAdminUser or logged=False. This needs to happen via nexus even if it means happening a bit later in startup.

Reply:
( and only if we are the active/leader )